content/docs/2.0/concepts/scaling-jobs.md

```diff
@@ -126,7 +126,7 @@ This logic is the same as Job for V1.  The number of the scale will be calculate
 _The number of the scale_
 
 ```go
-queueLength - runningJobCount
+maxScale - runningJobCount
```

Comment:
https://github.com/kedacore/keda/blob/aa0b7d4f126dd4fdc517e93711f4767bf21d6b73/pkg/scaling/scale_handler.go#L298

maxScale is not the running Job count. It is `maxValue = min(scaledJob.MaxReplicaCount(), devideWithCeil(queueLength, targetAverageValue))`  that means, pure the value queueLength devided by targetAvarageValue unless it is exceed the MaxReplicaCount.  You also wrote about this in the note. 

RunningJobCount is fetched in here. So that maxScale != RunningJobCount. It might be confusing. Probably, you can explain, RunnningJobCount , PendingJobCount, and  maxScale in here.   
 https://github.com/kedacore/keda/blob/aa0b7d4f126dd4fdc517e93711f4767bf21d6b73/pkg/scaling/executor/scale_jobs.go#L29
https://github.com/kedacore/keda/blob/aa0b7d4f126dd4fdc517e93711f4767bf21d6b73/pkg/scaling/executor/scale_jobs.go#L322

Reply:
Thanks, @TsuyoshiUshio for taking look at this,

Just for confirmation:
I have already written about maxScale in line no. 156 but I will modify it according to your suggestion.

For RunningJobCount and PendingJobCount is this right to write like this:
 
RunningJobCount means the number of jobs that are in the running state or not finished. It is measured through:
``` 
if !e.isJobFinished(&job) {
		runningJobs++		
}
```
PendingJobCount means the no. of jobs that are in pending state i.e job which is not finished  **AND** pod w.r.t job is either not running or not completed. it is measured through:
```
if !e.isJobFinished(&job) && !e.isAnyPodRunningOrCompleted(&job) {
			pendingJobs++
}
```
Please suggest any modification you want to it. and also please tell me where you want me to add these definitions in the docs.

Reply:
Sounds good! I think, the definition will be just after the maxScale might be fine Since the following code example includes RunningJobCount and PendingJobCount. So that the definition should come before these.

Reply:
Thanks! @TsuyoshiUshio 
Will soon create a PR for it.

Reply:
Hi @TsuyoshiUshio 
Addressed the comment. PTAL!

Reply:
Thanks! For me Good to go. 
I'm not very good at English, (especially this is doc) I'll ask someone else to review  as well.